### PR TITLE
Added Event Card component and displayed two upcoming events on home page

### PR DESCRIPTION
**Pull request by KaytonFletcher**
I do not know the details of the ultimate software event so they need to be added.

### CODE REVIEW

File: components/event.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                       
+                            {<FontAwesomeIcon className="mr-1" icon={faClock} fixedWidth />}{props.datetime}  
+		                    <FontAwesomeIcon className="ml-3 mr-1" icon={faMapPin} />{props.location} 
+	                 
+                    {/* <Button variant="primary">Go somewhere</Button> */}
+                </Card.Body>
+                </Card>
+            </Col>
+       </Row>
+    )
```
Comment (ConnorJC3) [inline components/event.js]: Inconsistent tabbing

File: components/config/event.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+
+const EVENT_DATA = [
+  {
+    "kind": "calendar#event",
+    "id": "jasjkfbakjfbjgbebzbcbau",
```
Comment (ConnorJC3) [inline components/config/event.js]: Junk data

File: components/config/event.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    "summary": "GBM 1",
+    "description": "Our first General Body Meeting of the Spring Semester. Come out for free pizza and to learn about upcoming ACE events!",
+    "location": "TBA",
+    "datetime": "1/15 6:30PM"
+  },
+  {
+    "summary": "Ultimate Software Event",
+    "description": "bruhhh",
```
Comment (ConnorJC3) [inline components/config/event.js]: Junk data

File: components/event.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+                    <Card.Title>{props.name}</Card.Title>
+                        <Card.Text>
+                         {props.description}
+                        </Card.Text>
+
+                       
+                            {<FontAwesomeIcon className="mr-1" icon={faClock} fixedWidth />}{props.datetime}  
+		                    <FontAwesomeIcon className="ml-3 mr-1" icon={faMapPin} />{props.location} 
+	                 
+                    {/* <Button variant="primary">Go somewhere</Button> */}
```
Comment (ConnorJC3) [inline components/event.js]: Remove commented code

File: package-lock.json [hunk 5]
```diff
@@ -0,0 +1,9075 @@
+{
```
Comment (ConnorJC3) [inline package-lock.json]: Remove this file from the diff (use yarn not npm)

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]

File: components/config/event.js (hunk 3 above)
Comment (KaytonFletcher) [reply components/config/event.js]: I edited the description, date and location for the ultimate event

File: components/config/event.js (hunk 2 above)
Comment (KaytonFletcher) [reply components/config/event.js]: Removed kind and id from event

File: package-lock.json (hunk 5 above)
Comment (KaytonFletcher) [reply package-lock.json]: Removed file from git

**Review by ConnorJC3 — APPROVED** [review]
~~lgtm~~
EDIT: see below, didn't realize this needs a rebase

**Review by ConnorJC3 — CHANGES_REQUESTED** [review]
Needs rebase before merge (is based on a super outdated HEAD)
Consider squashing commits before merge for repo size reasons

**Review by ConnorJC3 — APPROVED** [review]
lgtm, merge with squash/rebase please

**Pull request merged**